src/data/layer.rs

```diff
@@ -12,3 +12,14 @@ pub struct Layer {
     pub cache: bool,
     pub metadata: Table,
 }
+
+impl Layer {
+    pub fn new() -> Self {
+        Layer {
+            launch: defaults::r#false(),
+            build: defaults::r#false(),
+            cache: defaults::r#false(),
```

Comment:
Shouldn't we just use `bool` literals there? I think we have those `fn`'s for serde which does not support defaults from literals.

Reply:
yeah, I was thinking about that. I picked this in case we ever change the default but happy to switch to false.